.pre-commit-config.yaml

```diff
@@ -0,0 +1,8 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+-   repo: https://github.com/PyCQA/flake8
+    rev: '6.1.0'
+    hooks:
+    -   id: flake8
+exclude: '.git|examples/.*|tests/.*'
```

Comment:
Is there a reason to exclude `examples/` and `tests/`?

Reply:
There wasn't, it was an earlier test to make sure CI works. I will add them back in and check flake8 again.